sdcm/provision/azure/virtual_machine_provider.py

```diff
@@ -222,6 +226,22 @@ def _get_scylla_storage_profile(image_id: str, name: str, disk_size: Optional[in
             })
         return storage_profile
 
+    def run_command(self, name: str, command: str) -> Result:
+        if name not in self._cache:
+            raise AttributeError(f"Instance '{name}' does not exist in resource group '{self._resource_group_name}'")
+        LOGGER.debug("Running command '%s' on instance: %s", command, name)
+        command_object = RunCommandInput(command_id="RunShellScript", script=[command])
+        result = self._azure_service.compute.virtual_machines.begin_run_command(
+            self._resource_group_name, name, command_object).result()
+        LOGGER.debug("Finished running command '%s' on instance: %s: %s", command, name, result.value[0])
+        stdout, stderr = result.value[0].message.split('[stdout]\n')[1].split('\n[stderr]\n')[0:2]
```

Comment:
something wrong with this command, in the test you ran... it got `list index out of range`

Reply:
reboot command does not provide stderr/stdout when executed like this, fixed and retried the run